pkgs/development/python-modules/pwntools/default.nix

```
@@ -58,10 +60,20 @@ buildPythonPackage rec {
 
   doCheck = false; # no setuptools tests for the package
 
+  postFixup = ''
+    mkdir -p "$out/bin"
+    makeWrapper "${debugger}/bin/${debugger.pname}" "$out/bin/pwntools-gdb"
+  '';
+
+  installCheckPhase = ''
```

in buildPythonPackage, the `checkPhase` maps to the `installCheckPhase`
```suggestion
  checkPhase = ''
```

---

Is there a way to run any tests after the `fixupPhase`?

---

Oh interesting!
In the end, because the debugger's binary is wrapped anyway, there is no clear way to test that either `gdb` or `pwndbg` was used.

```shell
python -c 'import pwn; print(pwn.gdb.binary())' # points to the wrapped binary: $out/bin/pwntools-dbg
${debugger}/bin/pwntools-gdb                    # points to either `gdb` or `pwndbg` in the store
```